### PR TITLE
Adding Feature buttons for Hero Boards - MKC-768

### PR DESCRIPTION
**Pull request by Hannes7eicher**
## What This PR Changes
This PR implements new buttons inside the feature sections for the Hero Boards highlighting the main features. Currently, it depends on [@karlsoderby's branch](https://github.com/arduino/docs.arduino.cc/pull/973).

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/02.hero/boards/leonardo/features.md [hunk 1]
```diff
@@ -1,31 +1,20 @@
...
-The high Performance, low power AVR® 8-bit microcontroller.
-
-<FeatureLink title="Datasheet" url="https://content.arduino.cc/assets/Atmel-7766-8-bit-AVR-ATmega16U4-32U4_Datasheet.pdf" download blank/>
-</Feature>
-
-<Feature title="Built-in USB communication" image="usb">
+<Feature title="USB communication" image="usb">
 
-The ATmega32U4 has built-in USB communication that allows the Micro to appear as a mouse/keyboard on your machine.
+The Arduino Leonardo has built-in USB communication that allows the Micro to appear as a mouse/keyboard on your machine.
```
Comment (karlsoderby) [inline content/hardware/02.hero/boards/leonardo/features.md:11]: Can you add the links to the mouse/keyboard docs?
It is here:

- https://www.arduino.cc/reference/en/language/functions/usb/keyboard/
- https://www.arduino.cc/reference/en/language/functions/usb/mouse/

File: content/hardware/02.hero/boards/mega-2560/features.md [hunk 2]
```diff
@@ -1,32 +1,27 @@
...
 <FeatureList>
 
-<Feature title="ATmega2560" image="core">
-
-The high Performance, low power AVR® 8-bit microcontroller.
-
-<FeatureLink title="Datasheet" url="https://content.arduino.cc/assets/ATmega640-1280-1281-2560-2561-Datasheet-DS40002211A.pdf" download blank/>
-</Feature>
-
 <Feature title="EEPROM" image="mcu">
```
Comment (karlsoderby) [inline content/hardware/02.hero/boards/mega-2560/features.md:10]: Can you add a link here that goes to:
- https://docs.arduino.cc/learn/programming/eeprom-guide

File: content/hardware/02.hero/boards/uno-rev3-smd/features.md [hunk 3]
```diff
@@ -8,14 +8,6 @@ The "SMD" stands for surface-mount device, and the microcontroller (ATmega328p)
...
 
-<Feature title="ATmega328P" image="core">
-
-The classic high-performance, low-power AVR® microcontroller. 
-
-<FeatureLink title="Datasheet" url="https://content.arduino.cc/assets/Atmel-7810-Automotive-Microcontrollers-ATmega328P_Datasheet.pdf" download blank/>
-
-</Feature>
-
 <Feature title="EEPROM" image="mcu">
```
Comment (karlsoderby) [inline content/hardware/02.hero/boards/uno-rev3-smd/features.md:11]: Can you add a link here to EEPROM? 
- https://docs.arduino.cc/learn/programming/eeprom-guide

File: content/hardware/02.hero/boards/uno-rev3/features.md [hunk 4]
```diff
@@ -6,14 +6,6 @@ Arduino UNO is a microcontroller board based on the **ATmega328P**. It has 14 di
...
-
-The classic high-performance, low-power AVR® microcontroller. 
-
-<FeatureLink title="Datasheet" url="https://content.arduino.cc/assets/Atmel-7810-Automotive-Microcontrollers-ATmega328P_Datasheet.pdf" download blank/>
-
-</Feature>
-
 <Feature title="Replaceable chip" image="configurability">
 
 The ATmega328P can easily be replaced, as it is not soldered to the board.
```
Comment (karlsoderby) [inline content/hardware/02.hero/boards/uno-rev3/features.md:11]: Can you add a link to EEPROM?

https://docs.arduino.cc/learn/programming/eeprom-guide

File: content/hardware/02.hero/boards/zero/features.md [hunk 5]
```diff
@@ -8,23 +8,18 @@ bootloader programming.
...
+The Zero comes with a built-in debugger, which can be used with the Arduino IDE 2.0 version.
 
+  <FeatureLink variant="primary" title="Documentation" url="/tutorials/zero/debugging-with-zero"/>
 </Feature>
 
 <Feature title="Battery Connector" image="power">
 
 The Arduino Zero features a barrel plug connector, that works great with a standard 9V battery.
 
+  <FeatureLink variant="secondary" title="Library" url="https://www.arduino.cc/reference/en/libraries/arduino-low-power/"/>
```
Comment (karlsoderby) [inline content/hardware/02.hero/boards/zero/features.md:22]: I would not link "low power" features in the "battery connector" section. They are related but one is a hardware connector and the other is a library for optimizing the power consumption of the board ;) 

Can you instead add an additional feature, call it low power, and point it to that library?

You can use the following:
- **Title:** Low Power Support
- **Description:** This board can be optimized to reduce power consumption via the Arduino Low Power library

**Review by karlsoderby — CHANGES_REQUESTED** [review]
Left a couple of comments!

File: content/hardware/02.hero/boards/leonardo/features.md (hunk 1 above)
Comment (Hannes7eicher) [reply content/hardware/02.hero/boards/leonardo/features.md:11]: Added both as secondary buttons

**Review by karlsoderby — APPROVED** [review]

**Pull request merged**